bin/sl-deploy.js

```diff
@@ -56,16 +59,19 @@ while ((option = parser.getopt()) !== undefined) {
       local = true;
       break;
     default:
-      console.error('Invalid usage (near option \'%s\'), try `%s --help`.',
-        option.optopt,
-        $0);
+      console.error(GLB.t('msgInvalidUsage1', {
+        phOption: option.optopt,
+        phHelp: $0 + ' --help',
```

Comment:
Why did you decide to concatenate `--help` into the replacement, rather than have it be a fixed string in the message string?